langchain/scripts/create-entrypoints.js

```diff
@@ -336,6 +336,8 @@ const entrypoints = {
   "experimental/tools/pyinterpreter": "experimental/tools/pyinterpreter",
   // evaluation
   evaluation: "evaluation/index",
+  // smith (LangSmith Evaluation)
+  smith: "smith/index",
```

Comment:
This was the only change in this file right? The rest is just formatting?

Reply:
More philosophically, does all this really belong in LangChain.js? Could it just be part of the `langsmith` SDK?

Reply:
Could be its own package. Reason for being here is langsmith can't have a dep on langchain